sdks/go/pkg/beam/core/runtime/graphx/coder.go

```diff
@@ -366,7 +370,7 @@ func (b *CoderMarshaller) Add(c *coder.Coder) string {
 		return b.internBuiltInCoder(urnVarIntCoder)
 
 	default:
-		panic(fmt.Sprintf("Unexpected coder kind: %v", c.Kind))
+		panic(fmt.Sprintf("failed to marshal custom coder %v, unexpected coder kind: %v", *c, c.Kind))
```

Comment:
The other panics are getting capitalized (Failed vs failed), please capitalize for consistency.

Errors should always be lowercased due to the concatenation convention. Makes them easier to read.

Reply:
Capitalized them all for consistency. I was aware of errors being lowercased, but wasn't sure about panic messages.